opencti-platform/opencti-graphql/src/domain/xtm-hub.ts

```diff
@@ -65,3 +66,51 @@ export const checkXTMHubConnectivity = async (context: AuthContext, user: AuthUs
 
   return { status: newRegistrationStatus };
 };
+
+export const autoRegisterOpenCTI = async (context: AuthContext, input: AutoRegisterInput): Promise<string> => {
+  const xtmHubUrl = nconf.get('xtm:xtmhub_url');
+
+  const response = await fetch(`${xtmHubUrl}/graphql-api`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'XTM-Hub-Platform-Token': input.platform_token ?? '',
+      'XTM-Hub-Platform-Id': input.platform_id ?? ''
+    },
+    body: JSON.stringify({
+      query: `
+      mutation AutoRegisterPlatform($platform: PlatformInput!) {
+        autoRegisterPlatform(platform: $platform) {
+          success
+        }
+      }
+    `,
+      variables: {
+        platform: {
+          id: input.platform_id,
+          url: input.platform_url,
+          title: input.platform_title,
+          contract: input.platform_contract,
+          version: input.platform_version
```

Comment:
Overall I don't understand why we are taking information from input instead of taking current platform informations.